Sources/HTTP/Responder/Responder.swift

```diff
@@ -1,3 +1,83 @@
+//public protocol Responder {
+//    func respond(to request: Request) throws -> Response
+//}
```

Comment:
Is there a reason why this code is commented, if so, maybe write a comment describing why